### PR TITLE
fix: cache page content, not page menus

### PR DESCRIPTION
**Pull request by rjsparks**

### CODE REVIEW

File: ietf/templates/meeting/past.html [hunk 1]
```diff
@@ -2,11 +2,13 @@
 {# Copyright The IETF Trust 2015, All Rights Reserved #}
 {% load origin %}
 {% load ietf_filters static %}
+{% load cache %}
 {% block pagehead %}
     <link rel="stylesheet" href="{% static "ietf/css/list.css" %}">
 {% endblock %}
 {% block title %}Past Meetings{% endblock %}
 {% block content %}
+    {% cache 3600 pastmeetings %}
```
Comment (jennifer-richards) [inline ietf/templates/meeting/past.html:11]: Needs `{% endcache %}`

File: ietf/templates/meeting/proceedings_progress_report.html [hunk 2]
```diff
@@ -1,7 +1,8 @@
 {% extends "base.html" %}
-{% load ams_filters ietf_filters %}
+{% load ams_filters ietf_filters cache %}
 {% block title %}IETF {{ meeting.number }} Proceedings - Progress Report{% endblock %}
 {% block content %}
+    {% cache 3600 proceedings_progress_report meeting.number %}
```
Comment (jennifer-richards) [inline ietf/templates/meeting/proceedings_progress_report.html:5]: Needs {% endcache %}

**Review by jennifer-richards — CHANGES_REQUESTED** [review]
I think you need to close your `{% cache... %}` tags (though maybe they work without that??)

Edit: no, the tests failed b/c of that, so they're needed.

**Comment by rjsparks** [comment]
D'oh

**Review by jennifer-richards — APPROVED** [review]

**Pull request merged**